lib/src/models/redditor.dart

```diff
@@ -19,47 +19,93 @@ import 'package:draw/src/models/mixins/messageable.dart';
 import 'package:draw/src/models/multireddit.dart';
 import 'package:draw/src/models/submission.dart';
 
-class Redditor extends RedditorRef {
+/// A fully initialized class representing a particular Reddit user, also
+/// known as a Redditor.
+class Redditor extends RedditorRef with RedditBaseInitializedMixin {
   /// The amount of comment karma earned by the Redditor.
   int get commentKarma => data['comment_karma'];
 
   /// The time the Redditor's account was created.
-  double get created => data['created'];
-
-  /// Returns the object's fullname.
-  ///
-  /// A fullname is an object's kind mapping (i.e., t3), followed by an
-  /// underscore and the object's ID (i.e., t1_c5s96e0).
-  Future<String> get fullname async => data['fullname'];
+  DateTime get createdUtc => (data['created_utc'] == null)
```

Comment:
Could GetterUtils be used here?

Reply:
Yup, I totally forgot this file. Will do.